code/game/objects/structures/door_assembly.dm

```diff
@@ -11,6 +11,7 @@
 	var/mineral = null
 	var/obj/item/electronics/airlock/electronics = null
 	var/airlock_type = /obj/machinery/door/airlock //the type path of the airlock once completed
+	var/multi_tile = FALSE //! do we perform the extra checks required for multi-tile (large) airlocks
```

Comment:
```suggestion
	///  Do we perform the extra checks required for multi-tile (large) airlocks?
	var/multi_tile = FALSE
```
Preferably fix the positioning of those other comments too, and for bonus points document the undocumented vars, but I'm not going to force you as you didn't add those.

Reply:
Sure